services/order.py

```diff
@@ -0,0 +1,30 @@
+from typing import Optional
+
+from django.db import transaction
+from django.utils.timezone import now
+from db.models import Order, Ticket, User, MovieSession
+
+
+@transaction.atomic
+def create_order(tickets: list[dict],
+                 username: str,
+                 date: Optional[str] = None) -> Order:
+    order_date = date if date else now()
+    user_instance = User.objects.get(username=username)
```

Comment:
The `create_order` function should include error handling to manage potential `DoesNotExist` exceptions when retrieving the `User` object. Consider using a try-except block to handle the exception.